client/my-sites/site-card/index.js

```diff
@@ -0,0 +1,20 @@
+/**
+ * External dependencies
+ */
+
+import page from 'page';
+
+/**
+ * Internal Dependencies
+ */
+import { navigation, siteSelection } from 'my-sites/controller';
+import { makeLayout, render as clientRender } from 'controller';
+
+/**
+ * Style dependencies
+ */
+import './style.scss';
+
+export default function () {
+	page( '/site-card/:domain?', siteSelection, navigation, null, makeLayout, clientRender );
```

Comment:
Did we mean to put something in place of that `null`?

Reply:
That argument loads the main section (primary). `null` is wrong cause it is not a function. I tried 
```
( context, next ) => next()
```
but couldn't get the nudges / notices. Since it is a just a proposal I just skipped it. Will need to fix it though eventually! Forgot to put it in the Current Issues list. Thanks!